### PR TITLE
add: developer_dao

### PR DESCRIPTION
**Pull request by therealharpaljadeja**
Request to add Developer Dao to BoardRoom. 
Please contact light#2927 on discord for any further information.

### CODE REVIEW

File: protocols/developer_dao/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{ 
+	"cname": "developer_dao",
```
Comment (Rashmi-278) [inline protocols/developer_dao/index.json:2]: Hey ! Could you change cname and path to developerdao please? Without the underscore.

**Review by Rashmi-278 — CHANGES_REQUESTED** [review]

File: protocols/developer_dao/index.json (hunk 1 above)
Comment (therealharpaljadeja) [reply protocols/developer_dao/index.json]: Done!

File: protocols/developer_dao/index.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{ 
+	"cname": "developerdao",
+	"description": "A community of builders who believe in collective ownership of the internet.",
+	"path": "developer_dao",
```
Comment (Rashmi-278) [inline protocols/developer_dao/index.json:4]: Can you also change the path to the same ?

Comment (Rashmi-278) [reply protocols/developer_dao/index.json:4]: @therealharpaljadeja

**Review by Rashmi-278 — APPROVED** [review]

**Pull request merged**